decidim-proposals/app/helpers/decidim/proposals/admin/proposals_helper.rb

```diff
@@ -9,8 +9,8 @@ module Admin
       module ProposalsHelper
         # Public: A formatted collection of Meetings to be used
         # in forms.
-        def meetings_selected
-          @meetings_selected ||= @proposal.authors.pluck(:id) if @proposal.present?
+        def meetings_as_authors_selected
+          @meetings_as_authors_selected ||= @proposal.authors.pluck(:id) if @proposal.present?
```

Comment:
What if the proposal has authors that aren't meetings but they have the same id of a meeting? Wouldn't that be a problem?

Reply:
:confused:

Reply:
Fixed!